src/amber/environment/settings.cr

```diff
@@ -52,5 +76,73 @@ module Amber::Environment
       :encrypted_cookie
       end
     end
+
+    def logging
+      @_logging ||= Logging.new(@logging)
+    end
+  end
+
+  class LoggerBuilder
+    def self.logger(io, logging)
+      new(io, logging).logger
+    end
+
+    def initialize(io, logging)
+      Colorize.enabled = logging.color
+      @logger = Environment::Logger.new(io)
+      @logger.level = logging.severity
+      @logger.progname = "Server"
+      @logger.formatter = Logger::Formatter.new do |severity, datetime, progname, message, io|
+        if logging.time == true
+          io << datetime.to_s("%Y-%m-%d %I:%M:%S")
+          io << " "
+        end
```

Comment:
Is there an advantage or a reason to have it this way instead of:
```crystal 
io << datetime.to_s("%Y-%m-%d %I:%M:%S") << " " if logging.time == true
# If a newline is needed, could this work:
io << datetime.to_s("%Y-%m-%d %I:%M:%S") << "\n" if logging.time == true
io << severity << "\n" if logging.level == true
...
```

Reply:
I find the above easier to read and is practically the same